From cebd653b3fd692f51838457a0b5b203f1281714e Mon Sep 17 00:00:00 2001
From: Chen Jun <chenjun102@huawei.com>
Date: Fri, 14 Jan 2022 19:30:43 +0800
Subject: [PATCH] Revert "[Backport] ARM: BCM53016: Specify switch ports for
 Meraki MR32"
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: cebd653b3fd692f51838457a0b5b203f1281714e


hulk inclusion
bugzilla: 185877 https://gitee.com/openeuler/kernel/issues/I4DDEL

--------------------------------

This reverts commit 3573d9d69944e012d995753eab9415de244b7bc9.

the commit 568d94c5c9f0 ("ARM: BCM53016: Specify switch ports for Meraki
MR32") Introduce some warning:

 property has invalid length (4 bytes) (#address-cells == 2, #size-cells
 == 1)
 arch/arm/boot/dts/bcm53016-meraki-mr32.dtb: Warning
 (pci_device_bus_num): Failed prerequisite 'reg_format'
 arch/arm/boot/dts/bcm53016-meraki-mr32.dtb: Warning (i2c_bus_reg):
 Failed prerequisite 'reg_format'
 arch/arm/boot/dts/bcm53016-meraki-mr32.dtb: Warning (spi_bus_reg):
 Failed prerequisite 'reg_format'
 arch/arm/boot/dts/bcm53016-meraki-mr32.dts:203.10-206.5: Warning
 (avoid_default_addr_size): /srab@18007000/ports/port@0: Relying on
 default #address-cells value
 arch/arm/boot/dts/bcm53016-meraki-mr32.dts:203.10-206.5: Warning
 (avoid_default_addr_size): /srab@18007000/ports/port@0: Relying on
 default #size-cells value
 arch/arm/boot/dts/bcm53016-meraki-mr32.dts:208.10-217.5: Warning
 (avoid_default_addr_size): /srab@18007000/ports/port@5: Relying on
 default #address-cells value
 arch/arm/boot/dts/bcm53016-meraki-mr32.dts:208.10-217.5: Warning
 (avoid_default_addr_size): /srab@18007000/ports/port@5: Relying on
 default #size-cells value

bcm53016-meraki-mr32.dtb will not be used, so revert it.

Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm/boot/dts/bcm53016-meraki-mr32.dts | 22 ----------------------
 1 file changed, 22 deletions(-)

diff --git a/arch/arm/boot/dts/bcm53016-meraki-mr32.dts b/arch/arm/boot/dts/bcm53016-meraki-mr32.dts
index 577a4dc604d9..612d61852bfb 100644
--- a/arch/arm/boot/dts/bcm53016-meraki-mr32.dts
+++ b/arch/arm/boot/dts/bcm53016-meraki-mr32.dts
@@ -195,25 +195,3 @@ partition4@800000 {
 		};
 	};
 };
-
-&srab {
-	status = "okay";
-
-	ports {
-		port@0 {
-			reg = <0>;
-			label = "poe";
-		};
-
-		port@5 {
-			reg = <5>;
-			label = "cpu";
-			ethernet = <&gmac0>;
-
-			fixed-link {
-				speed = <1000>;
-				duplex-full;
-			};
-		};
-	};
-};
-- 
2.26.2

